deployment/Install-Client.ps1

```diff
@@ -152,6 +152,15 @@ function Install-Client {
             throw 'Errors while creating Optimized Production Build'
         }
 
+        az storage logging update --log rwd --retention 30 --services b --account-name $StaticWebsiteStorageAccount --only-show-errors
+        # Checking if the command for enabling logs of storage account failed
+        if($LASTEXITCODE -ne 0){
+            $manualStepsURL = "https://docs.microsoft.com/en-us/azure/storage/common/storage-monitor-storage-account#configure-logging"
```

Comment:
I believe we've created an aka.ms link for the same. We might consider using that here.

Reply:
Done. I had created the link, yes, but I was under the impression we could host a separate link via `aka.ms` by modifying the content somewhere; just realized that it is primarily a redirection service. I have updated the link now.